packages/itwinui-css/backstop/backstop.js

```diff
@@ -31,7 +31,7 @@ const config = {
   debug: false,
   debugWindow: false,
   dockerCommandTemplate:
-    'docker run --rm -i --mount type=bind,source="{cwd}",target=/src itwinui/backstopjs:latest {backstopCommand} {args}',
+    'docker run --rm -i --mount type=bind,source="{cwd}",target=/src bentleysystemsinc/itwinui:backstop {backstopCommand} {args}',
```

Comment:
Is it correct? Because previously it was `:latest` instead of some version and that makes sense but now it has `:backstop`. What it means?

Reply:
I have changed things a bit and updated whole description accordingly. @mayank99 @bentleyvk 

@gretanausedaite I am building now multiarch image, check up again, if it passes on MAC please.

Reply:
Working for me on M1 mac 👍